templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Car</h1>
+
+  <p>Are you sure you want to delete the car: {{ car }}?</p>
+
+  <form action="" method="post">
+    {% csrf_token %}
+    <input type="submit" value="Yes, delete!" class="btn btn-danger">
```

Comment:
Just a hint: you can make margins between buttons in bootsrap adding something like "mr-2" to classes (margin - direction - amount). So an element with class="btn btn-danger ml-4" will have a margin of 4em to the left of it
It can easily make a page look better.

Reply:
Thanks, changed.